ckanext/xloader/jobs.py

```diff
@@ -314,12 +314,14 @@ def _download_resource(resource, data, api_key, logger):
 
 def get_response(url, headers):
     def get_url():
+        proxy = config.get('ckan.download_proxy')
```

Comment:
what happens if proxy is not set?

Reply:
It gets set to `None` and is ignored (that's tested).